src/pages/Products/Products.jsx

```diff
@@ -1,12 +1,24 @@
-import { Typography } from '@mui/material'
+import { Container } from '@mui/material'
 
-import Localizator from '../../common/components/Localizator'
+import ProductsLoading from './components/ProductsLoading'
+import ProductsError from './components/ProductsError'
 
 const Products = () => {
   return (
-    <Typography>
-      <Localizator str="Products" />
-    </Typography>
+    <>
+      <Container
+        sx={{
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'row',
+          flexWrap: 'wrap',
+        }}
+        maxWidth="lg"
+      >
+        <ProductsLoading />
```

Comment:
wrap with Stack component;
pass styles from Container's sx attribute to the Stack

info: [Stack](https://mui.com/material-ui/react-stack/#main-content )